src/guides/v2.3/graphql/queries/index.md

```diff
@@ -133,6 +133,299 @@ Variables are defined separately in JSON:
 }
 ```
 
+## Staging queries {#staging}
+
+Magento GraphQL allows you to use certain queries to return preview information for staged content. Staging, a {{site.data.var.ee}} feature, allows merchants to schedule a set of changes to the storefront that run for a prescribed time in the future. These changes, also known as a _campaign_, are defined within the Admin. Customers do not have access to staged content, and as a result, staging queries have requirements that do not apply to traditional queries and mutations.
+
+[Content Staging](https://docs.magento.com/m2/ee/user_guide/cms/content-staging.html) in the _Merchant User Guide_ describes how to create a campaign.
+
+You can use the following queries to return staged preview information.
+
+*  `categoryList`
+*  `products`
+
+{:.bs-callout-info}
+The `products` query does not support full text search in the context of staging, because staged content is not indexed. Therefore, omit the `search` input attribute in your staging `products` queries.
+
+A staging query requires two specialized headers:
+
+Header name | Description
+--- | ---
+`Authorization Bearer: <authorization_token>` | An admin token. Use the `GET /V1/integration/admin/token` REST endpoint to generate this token.
+`Preview-Version` | A timestamp (seconds since January 1, 1970) that is inside the range of dates of the campaign you are querying.
+
+Magento returns an authorization error if you specify an invalid token or do not include both headers. If the specified timestamp does not correspond to a date in a scheduled campaign, Magento returns value based on the current storefront settings.
```

Comment:
```suggestion
Magento returns an authorization error if you specify an invalid token or do not include both headers. If the specified timestamp does not correspond to a date in a scheduled campaign, Magento returns a value based on the current storefront settings.
```
better yet, can we be more specific about what magento returns? An error code? A default date/timestamp?

Reply:
Addressed this comment by rewriting the sentence. The queries aren't returning an error message or similar--they're returning standard results.